example/ionic-angular-v3/src/app/tab1/tab1.page.ts

```diff
@@ -126,20 +126,14 @@ export class Tab1Page {
   }
 
   public clearUser(): void {
-    Sentry.configureScope(scope => {
-      scope.setUser(null);
-    });
+    Sentry.setUser(null);
   }
 
   public clearBreadcrumbs(): void {
-    Sentry.configureScope(scope => {
-      scope.clearBreadcrumbs();
-    });
+    Sentry.getGlobalScope().clearBreadcrumbs();
```

Comment:
To keep the same behavior as in v7 we need to clear all the scopes.
```suggestion
    Sentry.getGlobalScope().clearBreadcrumbs();
    Sentry.getIsolationScope().clearBreadcrumbs();
    Sentry.getCurrentScope().clearBreadcrumbs();
```

Reply:
I believe that was the code generated by the migration tool.
Ideally we should expose clearBreadcrumbs on the SDK and then it clears on all 3 scopes, what do you think?